RELEASING.md

```diff
@@ -0,0 +1,19 @@
+# Releasing
+
+## Process
+
+For version v0.x.y:
+
+1. Create the annotated tag
+   > NOTE: To use your GPG signature when pushing the tag, use `SIGN_TAG=1 ./contrib/tag-release.sh v0.x.y` instead)
```

Comment:
I thought I asked this already, perhaps I have. Is there any technical reason we can't use GH's release UI?